test/odbc/psqlodbc/bigint.cpp

```diff
@@ -374,11 +374,12 @@ TEST_F(PSQL_DataTypes_BigInt, Table_Unique_Constraints) {
 }
 
 TEST_F(PSQL_DataTypes_BigInt, Table_Single_Primary_Keys) {
+  long long int data;
+
```

Comment:
If we're doing minor changes to other files not related to the title, please address these in the description for visibility (doesn't need to be too detailed, something simple like resolved some minor issues in bigint.cpp and real)